plugins/modules/autoscaling_group_info.py

```diff
@@ -76,166 +76,272 @@
+        service_linked_role_arn:
+            description: The ARN of the service-linked role that the Auto Scaling group uses to call other Amazon Web Services on your behalf.
+            returned: success
+            type: str
+            sample: "arn:aws:iam::721234567890:role/aws-service-role/autoscaling.amazonaws.com/AWSServiceRoleForAutoScaling"
+        suspended_processes:
+            description: The suspended processes associated with the group.
+            returned: success
+            type: list
+            elements: dict
+            sample: [{
+                        "process_name": "AddToLoadBalancer",
+                        "suspension_reason": "User suspended at 2018-08-30T14:11:58Z"
+                    }]
+            contains:
+                process_name:
+                    description: The name of the suspended process.
+                    type: str
+                suspension_reason:
+                    description: The reason that the process was suspended.
+                    type: str
+        status:
+            description: The current state of the group when DeleteAutoScalingGroup is in progress.
+            returned: success
+            type: str
+            sample: None
+        tags:
+            description: List of tags for the ASG, and whether or not each tag propagates to instances at launch.
+            returned: success
+            type: list
+            sample: [
+                {
+                    "key": "Name",
+                    "value": "public-webapp-production-1",
+                    "resource_id": "public-webapp-production-1",
+                    "resource_type": "auto-scaling-group",
+                    "propagate_at_launch": "true"
+                },
+                {
+                    "key": "env",
+                    "value": "production",
+                    "resource_id": "public-webapp-production-1",
+                    "resource_type": "auto-scaling-group",
+                    "propagate_at_launch": "true"
+                }
+            ]
+        target_group_arns:
+            description: List of ARNs of the target groups that the ASG populates
+            returned: success
+            type: list
+            sample: [
+                "arn:aws:elasticloadbalancing:ap-southeast-2:123456789012:targetgroup/target-group-host-hello/1a2b3c4d5e6f1a2b",
+                "arn:aws:elasticloadbalancing:ap-southeast-2:123456789012:targetgroup/target-group-path-world/abcd1234abcd1234"
+            ]
+        target_group_names:
+            description: List of names of the target groups that the ASG populates
+            returned: success
+            type: list
+            sample: [
+                "target-group-host-hello",
+                "target-group-path-world"
+            ]
+        termination_policies:
+            description: A list of termination policies for the group.
+            returned: success
+            type: str
```

Comment:
If it's a list, shouldn't type be list?

Reply:
fixed. missed fixing that, was supposed to be
```
type: list
elements: str
```